source/docs/getting-started/getting-started-frc-control-system/control-system-hardware.rst

```diff
@@ -7,6 +7,13 @@ The goal of this document is to provide a brief overview of the hardware compone
 
 .. note:: For complete wiring instructions/diagrams, please see the :doc:`Wiring the FRC Control System <how-to-wire-a-robot>` document.
 
+Overview of 2019 Control System
+-------------------------------
+
+.. image:: images/control-system-hardware/frc-control-system-team-3161.png
```

Comment:
Can we use a lower quality source image for this? 2.5MB is a lot.

Reply:
I downsized this image to 938 KB in 40c1081. Is this good?